README.md

```diff
@@ -1 +1,50 @@
-# puppetlabs-cron_core
\ No newline at end of file
+
+# cron_core
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Setup - The basics of getting started with cron_core](#setup)
+    * [What cron_core affects](#what-cron_core-affects)
+    * [Setup requirements](#setup-requirements)
+    * [Beginning with cron_core](#beginning-with-cron_core)
```

Comment:
A number of the sections in the ToC don't appear below in the README. Should probably delete them from the ToC.